<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1399</width>
    <height>956</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::ForceTabbedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="scenes">
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="documentMode">
          <bool>true</bool>
         </property>
         <property name="tabsClosable">
          <bool>true</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1399</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Scene</string>
    </property>
    <widget class="QMenu" name="menuNew_2">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionCustom"/>
     <addaction name="separator"/>
     <addaction name="actionGame"/>
     <addaction name="actionComic"/>
     <addaction name="actionPresentation"/>
     <addaction name="separator"/>
     <addaction name="actionGet_More_3"/>
     <addaction name="actionOrganize_3"/>
    </widget>
    <addaction name="menuNew_2"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuLayers">
    <property name="title">
     <string>Layer</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionFreehand"/>
     <addaction name="actionVectors"/>
     <addaction name="actionText"/>
     <addaction name="separator"/>
     <addaction name="actionGet_More"/>
     <addaction name="actionOrganize"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionRemove"/>
    <addaction name="separator"/>
    <addaction name="actionImport_2"/>
    <addaction name="actionExport_2"/>
    <addaction name="separator"/>
    <addaction name="actionProperties"/>
   </widget>
   <widget class="QMenu" name="menuLayout">
    <property name="title">
     <string>Layout</string>
    </property>
    <widget class="QMenu" name="menuViews">
     <property name="title">
      <string>Views</string>
     </property>
     <addaction name="actionProperties_2"/>
     <addaction name="actionTimeline"/>
     <addaction name="actionAssets"/>
     <addaction name="actionTools"/>
     <addaction name="separator"/>
     <addaction name="actionGet_More_2"/>
     <addaction name="actionOrganize_2"/>
    </widget>
    <widget class="QMenu" name="menuTabs">
     <property name="title">
      <string>Tabs</string>
     </property>
     <addaction name="actionTabsNorth"/>
     <addaction name="actionTabsWest"/>
     <addaction name="actionTabsEast"/>
     <addaction name="actionTabsSouth"/>
    </widget>
    <addaction name="actionStandard"/>
    <addaction name="actionCompact"/>
    <addaction name="separator"/>
    <addaction name="menuViews"/>
    <addaction name="menuTabs"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionPreview"/>
   </widget>
   <addaction name="menuImage"/>
   <addaction name="menuLayers"/>
   <addaction name="menuLayout"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionFreehand">
   <property name="text">
    <string>Freehand</string>
   </property>
  </action>
  <action name="actionVector_Assisted">
   <property name="text">
    <string>Vector Assisted</string>
   </property>
  </action>
  <action name="actionVectors">
   <property name="text">
    <string>Vectors</string>
   </property>
  </action>
  <action name="actionText">
   <property name="text">
    <string>Text</string>
   </property>
  </action>
  <action name="actionGet_More">
   <property name="text">
    <string>Get More...</string>
   </property>
  </action>
  <action name="actionOrganize">
   <property name="text">
    <string>Organize...</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionOpacity">
   <property name="text">
    <string>Opacity</string>
   </property>
  </action>
  <action name="actionStandard">
   <property name="text">
    <string>Standard</string>
   </property>
  </action>
  <action name="actionCompact">
   <property name="text">
    <string>Compact</string>
   </property>
  </action>
  <action name="actionTimeline">
   <property name="text">
    <string>Timeline</string>
   </property>
  </action>
  <action name="actionTools">
   <property name="text">
    <string>Tools</string>
   </property>
  </action>
  <action name="actionAssets">
   <property name="text">
    <string>Assets</string>
   </property>
  </action>
  <action name="actionProperties_2">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionExport_2">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionImport_2">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionGet_More_2">
   <property name="text">
    <string>Get More...</string>
   </property>
  </action>
  <action name="actionOrganize_2">
   <property name="text">
    <string>Organize...</string>
   </property>
  </action>
  <action name="actionCustom">
   <property name="text">
    <string>Custom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionComic">
   <property name="text">
    <string>Comic</string>
   </property>
  </action>
  <action name="actionPresentation">
   <property name="text">
    <string>Presentation</string>
   </property>
  </action>
  <action name="actionGame">
   <property name="text">
    <string>Game</string>
   </property>
  </action>
  <action name="actionGet_More_3">
   <property name="text">
    <string>Get More...</string>
   </property>
  </action>
  <action name="actionOrganize_3">
   <property name="text">
    <string>Organize...</string>
   </property>
  </action>
  <action name="actionPackage">
   <property name="text">
    <string>Package</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="text">
    <string>Preview</string>
   </property>
  </action>
  <action name="actionTabsNorth">
   <property name="text">
    <string>North</string>
   </property>
  </action>
  <action name="actionTabsWest">
   <property name="text">
    <string>West</string>
   </property>
  </action>
  <action name="actionTabsEast">
   <property name="text">
    <string>East</string>
   </property>
  </action>
  <action name="actionTabsSouth">
   <property name="text">
    <string>South</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>Window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>699</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
